NormalizModule.cpp

```diff
@@ -382,6 +382,19 @@ PyObject* NmzToPyNumber(renf_elem_class in)
 }
 #endif
 
+PyObject* NmzToPyNumber(const dynamic_bitset& in)
+{
+    PyObject* result = PyList_New(0);
+    for (int i = 0; i < in.size(); i++) {
+        if (in[i]) {
+            PyObject* number = NmzToPyNumber(i);
+            PyList_Append(result, number);
+            Py_DecRef(number);
```

Comment:
@sebasguts Is there a reason you are not using `NmzBitsetToPyList`, which returns a list of bools, but instead write this new function which returns a list of integers?